rkt/rkt.go

```diff
@@ -27,6 +27,7 @@ import (
 	"github.com/coreos/rkt/rkt/config"
 	rktflag "github.com/coreos/rkt/rkt/flag"
 	"github.com/spf13/cobra"
+	_ "github.com/spf13/cobra/doc" // transitive dependency for man page generation
```

Comment:
Or the vendoredApps route would be fine actually...


On Wed, Apr 6, 2016 at 2:44 PM Jonathan Boulle <jonathan.boulle@coreos.com>
wrote:

> (tag support landed relatively recently via
> https://github.com/tools/godep/issues/271 )
>
> On Wed, Apr 6, 2016 at 2:44 PM Jonathan Boulle <jonathan.boulle@coreos.com>
> wrote:
>
>> right, that's where I was going - let's do that!
>>
>> On Wed, Apr 6, 2016 at 2:42 PM Sergiusz Urbaniak <
>> notifications@github.com> wrote:
>>
>>> In rkt/rkt.go
>>> <https://github.com/coreos/rkt/pull/2382#discussion_r58697141>:
>>>
>>> > @@ -27,6 +27,7 @@ import (
>>> >  	"github.com/coreos/rkt/rkt/config"
>>> >  	rktflag "github.com/coreos/rkt/rkt/flag"
>>> >  	"github.com/spf13/cobra"
>>> > +	_ "github.com/spf13/cobra/doc" // transitive dependency for man page generation
>>>
>>> I use godep v60.
>>>
>>> Wait, I found something, godep explicitely ignores +ignore tags, if I
>>> use +doc tags in the _gen.go files, godep is happy.
>>>
>>> —
>>> You are receiving this because you were mentioned.
>>> Reply to this email directly or view it on GitHub
>>> <https://github.com/coreos/rkt/pull/2382/files/25f9cd2d2ca12f99738919fe4740e1c9823bba92..c62a59a4661fa1059bd696a17d4afbf84bdf0730#r58697141>
>>>
>>